### PR TITLE
Changes value of syndicate briefcase with cash from 5 to 3.

### PR DESCRIPTION
**Pull request by Helg2**
## About The Pull Request
Changes price of syndicate briefcase full of cash from 5 to 3 tc. Returns its price to the original one.
So i never saw anyone buy that thing, probably because of it's cost, you can buy a bag with x-4 for 4 tc and a mini-bomb for 6 tc, and 5k of cash for 5 tc? Like why? It's useless in most cases.
And you hardly ever use that much money, unless you want to order custom shuttle like Monastery or something like that, and even then you cant do it normally because of its price in 5 tc. 
And as kopoba mentioned in comments crab protocol is worth of 7 tc and gets you even bigger amount of money.
## Why It's Good For The Game
This item will be used at least. Sometimes...
## Changelog
:cl:
balance: Syndicate briefcase full of cash is now worth of 3 tc instead of 5. Better than nothing
/:cl:

### CODE REVIEW

**Comment by kopoba** [comment]
i recently discover that all money from crab 17 protocol goes to traitor account so you will get ~40k for 7 tc

**Comment by Iamgoofball** [comment]
talk to @ArcaneMusic first

**Comment by Ekaterina-von-Russland** [comment]
On top of the money itself, Crab-17 has the added effect of disrupting the crew, which a simple briefcase with money does not. It makes sense that the briefcase should be a lot cheaper.
Good change.

**Comment by san7890** [comment]
Pre-#65795 it was 1 TC = 5000 Cr 
Post-#65795 it's now 1 TC = 1000 Cr

Now, you propose to make it 1 TC = 2500 Cr

Not sure how balanced this is especially due to progression traitors... at least people can swipe their cards on the Crab-17 to stop giving you money (and you're inextricably linked to the station's wealth)

**Comment by Helg2** [comment]
> Pre-#65795 it was 1 TC = 5000 Cr Post-#65795 it's now 1 TC = 1000 Cr
> 
> Now, you propose to make it 1 TC = 2500 Cr
> 
> Not sure how balanced this is especially due to progression traitors... at least people can swipe their cards on the Crab-17 to stop giving you money (and you're inextricably linked to the station's wealth)

I meant like the original price when it was firstly added in-game, but i can't find the source now... Because 1 tc for money, and 1 for briefcase.

You cant swipe department budgets, except cargo. And there will always be ssd, which will bring you money, and it's a great distraction for people.

Can change it to 1 tc if you want though, not much to argue about.

**Comment by Helg2** [comment]
@Mothblocks hi

File: code/modules/uplink/uplink_items/badass.dm [hunk 1]
```diff
@@ -40,7 +40,7 @@
 			and services at lucrative prices. The briefcase also feels a little heavier to hold; it has been \
 			manufactured to pack a little bit more of a punch if your client needs some convincing."
 	item = /obj/item/storage/secure/briefcase/syndie
-	cost = 5
+	cost = 2
```
Comment (ArcaneMusic) [inline code/modules/uplink/uplink_items/badass.dm:43]: `	cost = 3`
Best compromise I can make, but I want to keep an eye on this to see if it get abused to hell. Sound good?

**Review by ArcaneMusic — CHANGES_REQUESTED** [review]
I was hoping for some feedback on my last comment but whatever

File: code/modules/uplink/uplink_items/badass.dm (hunk 1 above)
Comment (Helg2) [reply code/modules/uplink/uplink_items/badass.dm:43]: And still practice use of this money is low.

Comment (Helg2) [reply code/modules/uplink/uplink_items/badass.dm:43]: But 3 is better than 5 so i'll just reduce it at some point if nothing like you described will happen.

**Comment by Mothblocks** [comment]
arcanes the only one who should act on this pr

**Pull request merged**